### ISSUE TITLE
Incompatible with Angular 9?

### ISSUE THREAD

**Issue by uberspeck** [jesusbotella/ngx-lazy-load-images#27]
Is this library compatible with Angular 9? I'm getting the following build error...

```
ERROR in node_modules/ngx-lazy-load-images/src/ngx-lazy-load-images.module.d.ts:1:22 - error
NG6002: Appears in the NgModule.imports of BooksModule, but could not be resolved to an NgModule class

1 export declare class LazyLoadImagesModule {
                       ~~~~~~~~~~~~~~~~~~~~
```

**Comment by jumpinvestor** [comment]
Same - had to remove it from the code entirely before the project would serve / build.

**Comment by uberspeck** [comment]
Try https://github.com/tjoskar/ng-lazyload-image That library is still maintained and works well 👍

**Comment by jesusbotella** [comment]
Hi @uberspeck and @jumpinvestor !

Sorry about that. It is true that I have not had time lately to update this repository so that the module works with Angular 9. I’m gonna open a PR to update and publish the module.

Once that is ready, if you miss or need any feature, feel free to contribute to the repository.

**Comment by uberspeck** [comment]
Hi @jesusbotella! No worries, I know how life gets in the way!

**Comment by jesusbotella** [comment]
Thank you for your issue.
I have just released a new version with Angular 11 support, please upgrade your `ngx-lazy-load-images` dependency in package.json to `^2.0.0`.

If you encounter any problems while using it, please open a new issue.
